docs/whats-new.md

```diff
@@ -52,6 +52,13 @@ Target Release Date: Q1 2022.
 
 - The `WKBWriter` supports options within a `wkb` key, instead of at the top level.
 
+**@loaders.gl/textures**
+- Fixed issues:
+  - Overflow issue on extreme colors in color_distance().
+  - Explicitly disabling unaligned addressing in miniz and transcoder when UBSAN is enabled
+  - Disabling a needless optimization in transcoder causing ubsan issues 
+- Upgrade `basis_universal` libraries to [v1.16.3](https://github.com/BinomialLLC/basis_universal/releases/tag/1.16.3).
```

Comment:
I think this line is all we need. Of course if we also support experimental worker writers etc we may want to mention that (separate PR). @belom88

Reply:
Keep only this line.